lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1199,6 +1199,12 @@ def _create_deployment_parameters(
         )
         arm_parameters.use_ipv6 = self._azure_runbook.use_ipv6
 
+        # Override IPv6 setting if environment specifically requires IPv6
```

Comment:
The overrides of requirements should be defined in the runbook, and the runbook requirements are merged with the test requirements in `lisa_runner`. The Azure Platform shouldn't modify it, as that would disrupt the flow.